chart/values.yaml

```diff
@@ -99,3 +99,5 @@ affinity:
 crd:
   enabled: true
   keep: true
+
```

Comment:
LGTM ! Thanks. 
Can you add comment on what the extra args might be. We can merge then. 
Thanks again :)